docs/guides/featureflags-or-experiments.md

```diff
@@ -4,39 +4,62 @@ title: When to use Feature Flags vs. Experiments?
+- **Experiment**: Use this when you want to test hypotheses between multiple product variants.
+
+---
+
+## Key Differences
 
-With Statsig, you can create control and test groups, and compare these groups as an A/B test using either a feature flag (gate) or an experiment. 
-If you're wondering when to use a feature flag vs. an experiment, this guide is for you.  
+Here are a few important considerations when deciding between a feature gate or an experiment for your A/B test:
 
-## What do you need?
-In short, 
- - Want to measure the impact of a feature (and roll it out gradually)? Use a **Feature Gate**.
- - Want to test your hypotheses between product variants? Create an **Experiment**.
+1. **Number of Variants**:
+   - **Feature Gate**: Provides only two variants (feature on vs. feature off).
+   - **Experiment**: Can handle multiple variants for more complex A/B testing.
+   - **Important Note**: When viewing feature gate exposures, you'll see three groups: "Pass," "Fail," and "Fail-Not in Analysis." While all users are bucketed into "Pass" or "Fail," only a balanced portion of the "Fail" group is used for metric analysis. You can read more about this in our [balanced gates methodology](/feature-flags/view-exposures#gate-exposures).
 
-## What to expect?
-As you design your A/B test, here are three further details to consider:
- - A Feature Gate can offer only two variants (feature on vs. feature off); an Experiment can offer multiple variants
-      - NOTE that when you look at your Feature Gate exposures you'll see your exposures divided up into three groups- "Pass", "Fail", and "Fail- not in analysis". While all units will be bucketed into either Pass/ Fail, for the purposes of metric lifts analysis, we compare a matching % of your Fail group against your Pass group and do not use the remainder of the Fail group for analysis. You can read more about this "balanced gates" methodology [here](/feature-flags/view-exposures#gate-exposures).
- - A Feature Gate exposure check returns a boolean (true/false) to assign the user to either the control or test group; an Experiment exposure check returns a JSON config to help you configure your app experience based on the group that the user is assigned to
- - To ramp up, you will increase the Pass% in a Feature Gate, and increase the Allocation% in an Experiment. In both cases, users in each group will continue to receive their first assigned experience i.e. ramping up Pass% or Allocation% does not resalt (or reshuffle) users. 
+2. **Exposure Results**:
+   - **Feature Gate**: Returns a boolean (`true` or `false`) to assign users to the control (off) or test (on) group.
+   - **Experiment**: Returns a JSON config that helps configure your app experience for the user's assigned group.
+
+3. **Ramping Up**:
+   - **Feature Gate**: Increase the **Pass%** to gradually roll out a feature to more users.
+   - **Experiment**: Increase the **Allocation%** to add more users to your test.
```

Comment:
you can go up to 99 vs 1 percent with fg, you cant go past 50/50 with expeirments